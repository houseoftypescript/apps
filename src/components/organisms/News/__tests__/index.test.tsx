import { render } from '@testing-library/react';
import { News } from '../index';

describe('News', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <News loading={true} error={null} data={[]} />
    );
    expect(container).toMatchSnapshot();
  });
});
