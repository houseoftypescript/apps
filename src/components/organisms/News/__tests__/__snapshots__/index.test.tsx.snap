// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News to match snapshot 1`] = `
<div>
  <div
    class="py-8 flex items-center justify-center"
  >
    <span
      class="uppercase text-white"
    >
      Loading
    </span>
  </div>
</div>
`;
